plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/AbstractFilesTable.java

```diff
@@ -31,15 +31,19 @@
 import io.trino.spi.type.TypeManager;
 import org.apache.iceberg.DataFile;
 import org.apache.iceberg.DataTask;
+import org.apache.iceberg.FileScanTask;
 import org.apache.iceberg.Schema;
 import org.apache.iceberg.StructLike;
 import org.apache.iceberg.Table;
 import org.apache.iceberg.TableScan;
+import org.apache.iceberg.io.CloseableIterable;
 import org.apache.iceberg.transforms.Transforms;
 import org.apache.iceberg.types.Conversions;
 import org.apache.iceberg.types.Type;
 import org.apache.iceberg.types.Types;
 
+import java.io.IOException;
+import java.io.UncheckedIOException;
```

Comment:
Change commit message `Wrapped scan.planFiles and task.rows in try-with-resource` to `Wrap scan.planFiles and task.rows in try-with-resource`

See https://cbea.ms/git-commit/#imperative

Reply:
updated the commit message.